### ISSUE TITLE
os.get_blocking call prevents Windows support, but sys.stdin, sys.stdout, sys.stderr is blocking by default on windows

### ISSUE THREAD

**Issue by doctorpangloss** [ansible/ansible#84283]
### Summary

I ought to be able to run `ansible-playbook` natively on Windows. One roadblock to compatibility is the call to `os.get_blocking` on each of `sys.stdin`, `sys.stderr` and `sys.stdout`. But those are all blocking on Windows, it is an unnecessary check. If they are pipes, then `os.get_blocking` ought to work - however it's not defined. It's safe to assume the `fd` as presented on Windows is blocking.

### Issue Type

Bug Report

### Component Name

ansible-playbook

### Ansible Version

```console
ansible==10.5.0
ansible-compat==24.9.1
ansible-core==2.17.5
ansible-lint==6.8.7
```

### Configuration

```console
(These are blocked on Windows)
```

### OS / Environment

Windows

### Steps to Reproduce

```yaml (paste below)
$ ansible-config dump --only-changed -t all
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "...\.venv\Scripts\ansible-config.exe\__main__.py", line 5, in <module>
  File "...\.venv\Lib\site-packages\ansible\cli\__init__.py", line 40, in <module>
    check_blocking_io()
  File "...\.venv\Lib\site-packages\ansible\cli\__init__.py", line 32, in check_blocking_io
    if not os.get_blocking(fd):
           ^^^^^^^^^^^^^^^
AttributeError: module 'os' has no attribute 'get_blocking
```

### Expected Results

This should run on Windows.

### Actual Results

```console
It does not.
```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

**Comment by ansibot** [comment]
Files identified in the description:

* [`bin/ansible-playbook`](https://github.com/ansible/ansible/blob/devel/bin/ansible-playbook)

If these files are incorrect, please update the `component name` section of the description or use the [component](https://github.com/ansible/ansibotmini#commands) bot command.

**Comment by sivel** [comment]
Duplicate of https://github.com/ansible/ansible/issues/79633

Additionally, per the [documentation](https://docs.ansible.com/ansible-core/devel/installation_guide/intro_installation.html#control-node-requirements), Windows is not a valid control system OS.
